src/include/miopen/fusion/solvers.hpp

```diff
@@ -277,6 +277,40 @@ struct ConvBinWinogradRxSf2x3g1Fused final : FusionSolverBase
     GetSolution(const FusionContext& context, const FusionDescription& problem) const override;
 };
 
+template <uint32_t Winodata, uint32_t Winofilter>
+struct ConvWinoFuryRxSFused final : FusionSolverBase
+{
+    const std::string& SolverDbId() const override
```

Comment:
Add `MIOPEN_INTERNALS_EXPORT` here. It is required for explicit template instantiation below (line 307).

Reply:
This templated solver structure is the only one fully instantiated at line 307. The rest of the solvers are partially instantiated templated structures.